README.md

```diff
@@ -18,26 +19,36 @@ You can obtain a binary from https://github.com/uw-labs/strongbox/releases
 Alternatively, assuming you have a working [Go](https://golang.org) installation, you can
 install via the following command:
 
-```bash
-go install github.com/uw-labs/strongbox@v1.1.0
+```console
+$ go install github.com/uw-labs/strongbox@v1.1.0
 ```
 
 ### Homebrew
 
-If you're on macOS or Linux and have [Homebrew](https://brew.sh/) installed, getting Strongbox is as simple as running:
+If you're on macOS or Linux and have [Homebrew](https://brew.sh/) installed,
+getting Strongbox is as simple as running:
 
-```
-brew install uw-labs/tap/strongbox
+```console
+$ brew install uw-labs/tap/strongbox
 ```
 
 ## Usage
 
+Strongbox supports [age](https://github.com/FiloSottile/age) and
+[siv](https://pkg.go.dev/github.com/jacobsa/crypto/siv?utm_source=godoc)
+encryption. Age is the recommended option.
+
+| encryption | identity / keyring file | recipient / key file |
+| ---------- | ----------------------- | -------------------- |
+| age        | .strongbox_identity     | .strongbox_recipient |
+| siv        | .strongbox-keyring      | .strongbox-keyid     |
```

Comment:
Maybe we should clarify that if both `.strongbox-keyid ` and `.strongbox_recipient` are present the latter takes precedence and the files will be age encrypted.